### PR TITLE
Update Mozilla Hubs logo (#12399)

### PR DESCRIPTION
**Pull request by yzAlvin**
## One-line summary

Update the hubs logo to the new one linked in the issue.

## Significant changes and points to review

changed the code to match patterns used for the other logos (referencing a separate svg file)

## Issue / Bugzilla link

#12399

## Screenshots

Screen Shot 2022-11-23 at 12 27 24 pm

## Checklist

If relevant:

- [ ] Tests added
- [ ] Feature flags added to www-config
- [ ] New secrets added to the secrets repository
- [ ] If new dependencies are added, I've checked their license is appropriate

## Testing

run locally and check icon under `innovation` dropdown

### CODE REVIEW

**Review by maureenlholland — CHANGES_REQUESTED** [review]
Looks great! Thanks for the PR. One tiny nit before approval and merge.

File: bedrock/base/templates/includes/protocol/navigation/menus/innovation.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
...
-                <svg class="c-menu-item-icon" xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32"><linearGradient id="a" x1="17.2471" x2="17.2471" y1="30" y2="3.1989" gradientTransform="matrix(1 0 0 -1 0 34)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ffc000"></stop><stop offset="1" stop-color="#ffd600"></stop></linearGradient><path fill="url(#a)" d="M7.5 24.7c-.3.8-1 1.5-1.9 2-.9-1.3-1.4-2.8-1.4-4.5 0-3.4 2-6.4 4.9-7.7-2.8-1.1-5.4-2.8-5.4-4.6 0-1.9 1.7-1.5 3.9-1.1h.1c.6-2.8 3.1-4.9 6-4.9 3.4 0 6.2 2.8 6.2 6.2 0 2.2-1.1 4.1-2.8 5.2 1.5.5 3.3.9 5.5.7 3-.4 4-2 4.6-2.9.3-.5.5-.8.8-.6 2.9 2.2 3.8 7.5 1.7 11.7-.5.9-1 1.7-1.7 2.4-.8 0-2-2-2-2-.2.6-2.3 2.7-4.6 2.7-2.1 0-4-1.1-4.6-2.7-.7 1.6-2.5 2.7-4.6 2.7-2.2.1-4.1-1-4.7-2.6"></path><circle cx="11.3" cy="9.6" r="1.7" transform="matrix(.9997 -.02585 .02585 .9997 -.2454 .2941)"></circle><path fill="#ff8a00" d="M7.9 8.3c-.1.2-.1.4-.2.6-2.2-.5-4-.9-4 1 0 1.8 2.6 3.5 5.4 4.6.2-3.2.3-4.6-1.2-6.2z"></path><circle cx="11.8" cy="9.2" r=".5" fill="#fff"></circle></svg>
+                <img class="c-menu-item-icon" src="{{ static('img/nav/icons/icon-mozilla-hubs.svg') }}" alt="" loading="lazy">
```
[the remark this answers is not in the document]
Comment (maureenlholland) [reply bedrock/base/templates/includes/protocol/navigation/menus/innovation.html:19]: @yzAlvin thanks again for the PR, are you available to make this update?

[the remark this answers is not in the document]
Comment (yzAlvin) [reply bedrock/base/templates/includes/protocol/navigation/menus/innovation.html]: hi @maureenlholland sorry for the late reply, have been away from computer for some time, thanks for the comment, have made the change

**Review by maureenlholland — APPROVED** [review]
💯 nice work @yzAlvin !

**Pull request merged**